packages/xrpl/src/client/index.ts

```diff
@@ -588,7 +602,13 @@ class Client extends EventEmitter {
    * @category Network
    */
   public async connect(): Promise<void> {
-    return this.connection.connect()
+    return this.connection.connect().then(async () => {
```

Comment:
I think this shouldn't fail if the network id call doesn't work. I'd suggest adding a `catch` to log an error but still allow the connection to succeed